authenticator/src/main/java/org/openmbee/sdvc/authenticator/security/JwtAuthenticationTokenFilter.java

```diff
@@ -42,7 +43,8 @@ public void doFilterInternal(HttpServletRequest request, HttpServletResponse res
                 String username = jwtTokenGenerator.getUsernameFromToken(authToken);
 
                 if (username != null && SecurityContextHolder.getContext().getAuthentication() == null) {
-                    UserDetailsImpl userDetails = this.userDetailsService.loadUserByUsername(username);
+                    //possible to get authorities from jwt instead of looking up in db also, if authorities were included in jwt
+                    UserDetails userDetails = this.userDetailsService.loadUserByUsername(username);
```

Comment:
if jwt doesn't contain authorities, may want separate modules for different user detail service impls and autowire the interface here instead of UserDetialsServiceImpl